addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -116,6 +116,7 @@
                                 <IdCodice t-esc="'OO99999999999'"/>
                             </IdFiscaleIVA>
                             <IdFiscaleIVA t-if="not record.commercial_partner_id.vat and record.commercial_partner_id.country_id.code != 'IT'">
+                                <IdPaese t-esc="record.commercial_partner_id.country_id.code"/>
```

Comment:
What if there is no country set on the partner? Is it possible or is there a check for that before? Then it will output `False`

Reply:
I did that before, but I considered that it is not done elsewhere in the file and when you confirm an invoice it will check if the country is set on the partner.